### PR TITLE
New format for custom HTTP headers

### PR DESCRIPTION
**Pull request by codebien**
Closes https://github.com/grafana/xk6-output-prometheus-remote/issues/101

### CODE REVIEW

File: pkg/remotewrite/config.go [hunk 1]
```diff
@@ -237,6 +237,19 @@ func parseEnvs(env map[string]string) (Config, error) {
 		c.Headers[k] = v
 	}
 
+	if headers, headersDefined := env["K6_PROMETHEUS_RW_HTTP_HEADERS"]; headersDefined {
+		if c.Headers == nil {
+			c.Headers = make(map[string]string)
```
Comment (olegbespalov) [inline pkg/remotewrite/config.go:242]: How about moving this into the initialization to the top by default:

```diff
-var c Config
+c := Config{Headers: make(map[string]string)}
```

There is no significant difference in efficiency, but the code's reader will be a bit happier without 
`if c.Headers == nil`

File: pkg/remotewrite/config.go [hunk 2]
```diff
@@ -237,6 +237,19 @@ func parseEnvs(env map[string]string) (Config, error) {
...
+		}
+		for _, kvPair := range strings.Split(headers, ",") {
+			header := strings.Split(kvPair, ":")
+			if len(header) != 2 {
+				continue
```
Comment (olegbespalov) [inline pkg/remotewrite/config.go:247]: Maybe we could log a warning that something is ignored (this `contunue` happening), it would be better for the UX

**Review by olegbespalov — COMMENTED** [review]
Looks good, left a two minor discussable comments

File: pkg/remotewrite/config.go (hunk 2 above)
Comment (codebien) [reply pkg/remotewrite/config.go]: I thought we could just return an error and avoid adding the logger dependency.

**Review by olegbespalov — APPROVED** [review]
:+1:

File: pkg/remotewrite/config.go (hunk 2 above)
Comment (olegbespalov) [reply pkg/remotewrite/config.go]: Yeah, also works since it attracts the user attention :+1:

**Review by mstoykov — APPROVED** [review]

**Pull request merged**